docs/src/paradox/index.md

```diff
@@ -70,20 +63,16 @@ object Demo extends IOApp {
 }
 ```
 
-Another common `Inject` instance we use is an encoder / decoder based on the Circe library:
+### Schema
+
+Neutron relies on `cats.Inject[A, Array[Byte]]` instances to be able to decode & encode messages from & to raw bytes instead of using the native schema solution. As functional programmers, we believe this has certain benefits. In the example above, we are using a standard "UTF-8" encoding of `String => Array[Byte]`, brought by `import cr.pulsar.schema.utf8._`.
+
+At ChatRoulette, we use JSON-serialised data for which we define an `Inject` instance based on Circe codecs. Those interested in doing the same can leverage the Circe integration by adding the `neutron-circe` extra dependency (available since `v0.0.2`).
```

Comment:
```suggestion
At Chatroulette, we use JSON-serialised data for which we define an `Inject` instance based on Circe codecs. Those interested in doing the same can leverage the Circe integration by adding the `neutron-circe` extra dependency (available since `v0.0.2`).
```